390/Serena_Li.Rmd

```diff
@@ -0,0 +1,42 @@
+---
+title: "MP2 Viz"
+author: "Serena Li"
+date: "9/15/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
+library(tidyverse)
+library(tsibble)
+library(lubridate)
+library(fpp3)
+library(dplyr)
+library(readr)
+```
+
+```{r}
+stem <- read.csv("scbi.stem1.csv")
+```
+
+```{r}
+avg_dbh<-stem %>% 
+  filter(!is.na(dbh),!is.na(sp)) %>% 
+  mutate(dbh_num=as.numeric(as.character(dbh))) %>% 
+  group_by(sp) %>% 
+  mutate(count=n(),
+           avg_dbh=sum(dbh_num/count))
+View(avg_dbh)
+```
+
+#barplot
+```{r}
+ggplot(avg_dbh,aes(x = factor(sp), y=avg_dbh)) +
```

Comment:
Great job! But these barplots only show the average i.e. the center of the distributions of dbh for each species. Why not show the spread as well? For example, do certain species have more *variation* in dbh than others. You can do this with a boxplot!

Reply:
Note: boxplots require several values of `y` for each value of `x`.

Reply:
In this case, you are plotting only a single value for each species: the average